readme.txt

```diff
@@ -33,6 +33,9 @@ You must have Genesis (2.4.2+) or a Genesis child theme installed and activated
 
 == Changelog ==
 
+= 2.2.2 =
+* Add compatibility with WordPress Coding Standards
```

Comment:
In readme.txt, some of these have a `.` at the end of the item, and some don't. We should add it to those that don't have it for consistency.

Reply:
Great catch @dreamwhisper should be resolved in the latest commit.